packages/backend/test/events/post-event.test.ts

```diff
@@ -0,0 +1,719 @@
+import dotenv from 'dotenv-flow'
+import { resolve } from 'path'
+import { Client } from 'pg'
+import t from 'tap'
+
+import createServer from '../../src/app'
+
+dotenv.config({
+  path: resolve(__dirname, '..', '..')
+})
+
+const client = new Client({
+  user: process.env.POSTGRES_USERNAME,
+  password: process.env.POSTGRES_PASSWORD,
+  host: process.env.POSTGRES_HOTNAME,
+  port: Number(process.env.POSTGRES_PORT),
+  database: process.env.POSTGRES_DBNAME
+})
+
+void t.test('post event', async t => {
+  const app = createServer()
+
+  t.teardown(async () => {
+    await app.close()
+    await client.end()
+  })
+  try {
+    await client.connect()
+    await client.query('delete from "events" where user_username = \'postevent\'')
```

Comment:
username should match username below

Reply:
![image](https://user-images.githubusercontent.com/63894614/154387795-c022bb3c-4e7c-4712-bf3b-04add961f052.png)
that right?